airbyte-workers/src/main/java/io/airbyte/workers/process/KubePodProcess.java

```diff
@@ -580,4 +600,19 @@ public int exitValue() {
     return getReturnCode(podDefinition);
   }
 
+  private static ResourceRequirementsBuilder getResourceRequirementsBuilder(ResourceRequirements resourceRequirements) {
+    if (resourceRequirements != null) {
+      final Map<String, Quantity> resourceMap = new HashMap<>();
+      // if null then use unbounded resource allocation
+      if (!com.google.common.base.Strings.isNullOrEmpty(resourceRequirements.getCpu())) {
+        resourceMap.put("cpu", Quantity.parse(resourceRequirements.getCpu()));
+      }
+      if (!com.google.common.base.Strings.isNullOrEmpty(resourceRequirements.getMemory())) {
+        resourceMap.put("memory", Quantity.parse(resourceRequirements.getMemory()));
+      }
+      return new ResourceRequirementsBuilder().withLimits(resourceMap);
```

Comment:
On this PR, I am just exposing the "limits" (not the requests parts) for the moment, I'm not sure if that's enough as a start? or should both be configurable?